opm/input/eclipse/Schedule/MSW/WellSegments.cpp

```diff
@@ -43,6 +44,7 @@
 #include <unordered_set>
 #include <utility>
 #include <vector>
+#include <iostream>
```

Comment:
This header (`<iostream>`) appears to be unused.  In that case, I'd prefer that we not include it.

Reply:
It is leftover when debugging, removed it now.